### PR TITLE
{bio}[iccifort/2019.5.281] BSseeker2 v2.1.8, Bowtie v1.2.3, Bowtie2 v2.3.5.1 w/ Python 2.7.16

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
gligar04.gastly.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6130 CPU @ 2.10GHz, Python 3.6.8
See https://gist.github.com/7ab371872bfbc21f46aee9b784c75169 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2434.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/dacbb51cffb63c4b46096b146efd9b34 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
generoso-1 - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/19c17697851b298a203a45613ac4b565 for a full test report.

File: easybuild/easyconfigs/b/BSseeker2/BSseeker2-2.1.8-GCC-8.3.0-Python-2.7.16.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pellegrini-legacy.mcdb.ucla.edu/bs_seeker2'
+description = """BS-Seeker2 is a seamless and versatile pipeline for accurately and fast mapping the bisulfite-treated
+reads."""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+source_urls = ['https://github.com/BSSeeker/BSseeker2/archive/']
+sources = ['BSseeker2-v%(version)s.tar.gz']
```
Comment (smoors) [inline easybuild/easyconfigs/b/BSseeker2/BSseeker2-2.1.8-GCC-8.3.0-Python-2.7.16.eb]:
```suggestion
sources = ['%(name)s-v%(version)s.tar.gz']
```

File: easybuild/easyconfigs/b/BSseeker2/BSseeker2-2.1.8-iccifort-2019.5.281-Python-2.7.16.eb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pellegrini-legacy.mcdb.ucla.edu/bs_seeker2'
+description = """BS-Seeker2 is a seamless and versatile pipeline for accurately and fast mapping the bisulfite-treated
+reads."""
+
+toolchain = {'name': 'iccifort', 'version': '2019.5.281'}
+
+source_urls = ['https://github.com/BSSeeker/BSseeker2/archive/']
+sources = ['BSseeker2-v%(version)s.tar.gz']
```
Comment (smoors) [inline easybuild/easyconfigs/b/BSseeker2/BSseeker2-2.1.8-iccifort-2019.5.281-Python-2.7.16.eb]:
```suggestion
sources = ['%(name)s-v%(version)s.tar.gz']
```

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 6 out of 6 (4 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/882cfb6d10ca18cf73ac2a33702edc1b for a full test report.

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**